templates/dashd.conf.template

```diff
@@ -39,16 +39,7 @@ zmqpubrawtxlocksig=tcp://0.0.0.0:29998
 [test]
 {{?? it.network === 'local'}}
 regtest=1
-[regtest]{{?? it.network === 'evonet'}}
```

Comment:
Let's keep it for network `devnet` in case if users decide to create a node for their network.

Reply:
OK, kept it in there and allowed `devnet` network types again. Do you want to add a system config for devnets as well? I think since it won't work with setup, we can just expect the user to configure a devnet manually.

Reply:
Yeah, exactly!